pkg/cmd/metrics.go

```diff
@@ -0,0 +1,19 @@
+package cmd
```

Comment:
What do you think about grouping all `*_metrics.go` under this single file, for easier access and readability? Do we need one subcommand per file?

Alternative: Put all in a `cmd/metrics/*` sub-directory

Reply:
There might be subcommands of subcommands too. So I just put each logical top-level separate command in a separate file here. 🙂

Reply:
Hm then how about the other suggestion with sub-directories? That way we could nicely represent the relationship between parent-children commands.

Reply:
I looked into this a bit more, about how cobra-based CLIs are organized on other projects (e,g kubectl/skaffold). Let's actually go with the single `metrics.go` file approach for now! Creating subdirs would require new packages for each top-level command and make sharing stuff like `logger` much harder.

And instead of having each command as a global variable with `init` functions to add flags (as suggested by cobra), it'd be better to add functions for each new command and use that (like in other projects). Makes it much more organized! 🙂

Reply:
Sounds good, thanks for addressing this!